internal/model/release_values.go

```diff
@@ -0,0 +1,250 @@
+}
+type Selector struct {
+	StorageClassName string           `yaml:"storageClassName,omitempty"`
+	AccessModes      []string         `yaml:"accessModes,omitempty"`
+	Requests         Requests         `yaml:"requests,omitempty"`
+	SelectorTemplate SelectorTemplate `yaml:"selectorTemplate,omitempty"`
+}
+type DefaultStorageClass struct {
+	AccessModes []string `yaml:"accessModes,omitempty"`
+	Requests    Requests `yaml:"requests,omitempty"`
+}
+type Dynamic struct {
+	StorageClassName string   `yaml:"storageClassName,omitempty"`
+	AccessModes      []string `yaml:"accessModes,omitempty"`
+	Requests         Requests `yaml:"requests,omitempty"`
+}
+type Volume struct {
+	SetOwnerAndGroupWritableFilePermissions bool `yaml:"setOwnerAndGroupWritableFilePermissions,omitempty"`
+}
+type VolumeClaimTemplate struct {
+}
+type Data struct {
+	Mode                string              `yaml:"mode,omitempty"`
+	Selector            Selector            `yaml:"selector,omitempty"`
+	DefaultStorageClass DefaultStorageClass `yaml:"defaultStorageClass,omitempty"`
+	Dynamic             Dynamic             `yaml:"dynamic,omitempty"`
+	Volume              Volume              `yaml:"volume,omitempty"`
+	VolumeClaimTemplate VolumeClaimTemplate `yaml:"volumeClaimTemplate,omitempty"`
+}
+type Share struct {
+	Name string `yaml:"name,omitempty"`
+}
+type Backups struct {
+	Mode  string `yaml:"mode,omitempty"`
+	Share Share  `yaml:"share,omitempty"`
+}
+type Logs struct {
+	Mode  string `yaml:"mode,omitempty"`
+	Share Share  `yaml:"share,omitempty"`
+}
+type Metrics struct {
+	Mode  string `yaml:"mode,omitempty"`
+	Share Share  `yaml:"share,omitempty"`
+}
+type Import struct {
+	Mode  string `yaml:"mode,omitempty"`
+	Share Share  `yaml:"share,omitempty"`
+}
+type Licenses struct {
+	Mode  string `yaml:"mode,omitempty"`
+	Share Share  `yaml:"share,omitempty"`
+}
+type Volumes struct {
+	Data     Data     `yaml:"data,omitempty"`
+	Backups  Backups  `yaml:"backups,omitempty"`
+	Logs     Logs     `yaml:"logs,omitempty"`
+	Metrics  Metrics  `yaml:"metrics,omitempty"`
+	Import   Import   `yaml:"import,omitempty"`
+	Licenses Licenses `yaml:"licenses,omitempty"`
+}
+type NodeSelector struct {
+}
+type Annotations struct {
+}
+type Default struct {
+	Annotations map[string]string `yaml:"annotations,omitempty"`
+}
+type Spec struct {
+	Type string `yaml:"type,omitempty"`
+}
+type HTTPPort struct {
+	Enabled bool `yaml:"enabled,omitempty"`
+}
+type HTTPSPort struct {
+	Enabled bool `yaml:"enabled,omitempty"`
+}
+type BoltPort struct {
+	Enabled bool `yaml:"enabled,omitempty"`
+}
+type BackupPort struct {
+	Enabled bool `yaml:"enabled,omitempty"`
+}
+type Ports struct {
+	HTTP   HTTPPort   `yaml:"http,omitempty"`
+	HTTPS  HTTPSPort  `yaml:"https,omitempty"`
+	Bolt   BoltPort   `yaml:"bolt,omitempty"`
+	Backup BackupPort `yaml:"backup,omitempty"`
+}
+type ServiceSelector struct {
+	HelmNeo4JComNeo4JLoadbalancer string `yaml:"helm.neo4j.com/neo4j.loadbalancer,omitempty"`
```

Comment:
can we change the name HelmNeo4JComNeo4JLoadbalancer to something more easy to read?

Reply:
type ServiceSelector struct { --> this should be a map[string]string with the below as default value
"helm.neo4j.com/neo4j.loadbalancer": "include"

Reply:
i dont see this serviceSelector in 4.4